front/app/modules/commercial/idea_custom_fields/admin/containers/projects/edit/ideaform/IdeaCustomField.tsx

```diff
@@ -234,101 +163,90 @@ export default memo<Props>(
-                          />
-                        }
-                      />
-                    </ToggleContainer>
-                  )}
-                  {fieldEnabled && canSetRequired && (
-                    <ToggleContainer>
-                      <StyledToggle
-                        checked={fieldRequired}
-                        onChange={handleRequiredOnChange}
-                        label={<FormattedMessage {...messages.required} />}
-                        labelTextColor={colors.adminTextColor}
-                        className={`
+                    />
+                    <IconTooltip
+                      content={
+                        <FormattedMessage {...messages.enabledTooltipContent} />
+                      }
+                    />
+                  </ToggleContainer>
+                )}
+                {fieldEnabled && canSetRequired && (
+                  <ToggleContainer>
+                    <StyledToggle
+                      checked={fieldRequired}
+                      onChange={handleRequiredOnChange}
+                      label={<FormattedMessage {...messages.required} />}
+                      labelTextColor={colors.adminTextColor}
+                      className={`
                           e2e-${ideaCustomField.attributes.code}-required-toggle-label
                       `}
-                      />
-                      <IconTooltip
-                        content={
-                          <FormattedMessage
-                            {...messages.requiredTooltipContent}
-                          />
-                        }
-                      />
-                    </ToggleContainer>
-                  )}
-                </Toggles>
-
-                {fieldEnabled && (
-                  <Suspense fallback={<Spinner />}>
-                    <QuillMutilocWithLocaleSwitcher
-                      id={`${ideaCustomField.id}-description`}
-                      noImages={true}
-                      noVideos={true}
-                      noAlign={true}
-                      valueMultiloc={descriptionMultiloc}
-                      onChange={handleDescriptionOnChange}
-                      label={
-                        <LocaleSwitcherLabelText>
-                          <FormattedMessage {...messages.descriptionLabel} />
-                        </LocaleSwitcherLabelText>
+                    />
+                    <IconTooltip
+                      content={
+                        <FormattedMessage
+                          {...messages.requiredTooltipContent}
+                        />
                       }
                     />
-                  </Suspense>
+                  </ToggleContainer>
                 )}
-              </CollapseContainerInner>
-            </CollapseContainer>
-          </CSSTransition>
+              </Toggles>
+
+              {fieldEnabled && (
+                <Suspense fallback={<Spinner />}>
```

Comment:
I wonder if there's really value in code-splitting here?

Reply:
Probably not. I left it there as I wasn't sure why it was added.

Reply:
I think it's fine to delete 👍